devnet/substrate_alice.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM parity/polkadot:v0.9.10 as polkadot
+FROM golang:1.17
+
+ARG VERSION=v0.9.10
+ARG DD_API_KEY=somekey
+ENV DD_API_KEY=${DD_API_KEY}
+ARG METRICS_NAMESPACE=substrate.local.devnet
+
+RUN apt-get update
+
+RUN DD_AGENT_MAJOR_VERSION=7 DD_INSTALL_ONLY=true DD_SITE="datadoghq.com" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
+
+COPY --from=polkadot /usr/bin/polkadot /usr/bin/polkadot
+COPY devnet/chain/gssmr/genesis-raw.json genesis-spec.json
+
+RUN go run cmd/update-dd-agent-confd/main.go -n=${METRICS_NAMESPACE} -t=key:alice > /etc/datadog-agent/conf.d/openmetrics.d/conf.yaml
```

Comment:
what port is prometheus running on? `update-dd-agent-config` is currently expecting the prometheus endpoint to be `http://127.0.0.1:9876/metrics`.

Reply:
Changed substrate nodes to expose Prometheus at port 9876

Reply:
What port does substrate uses? Shouldn't we use the standard `9090` port for Gossamer as well 🤔

Reply:
by default the port 9090 is the one Prometheus docker image [prom/prometheus](https://hub.docker.com/r/prom/prometheus) uses to bootstrap it's own server to collect metrics

Reply:
Yes but by default what's the port substrate/polkadot uses for Prometheus? We force it to `9876` here it seems. I'm thinking we should change it in Gossamer (for end users experience).

As far as I know, applications usually have their Prom registry server listening on `9090`, so maybe we want to use that instead? Although if polkadot uses 9876 for whatever weird reason, then sure we can match it as well I guess.